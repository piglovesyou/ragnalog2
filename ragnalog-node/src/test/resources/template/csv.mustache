in:
  type: file
  path_prefix: {{input_file}}
  parser:
    charset: UTF-8
    newline: CRLF
    type: csv
    columns:
      - {name: timestamp, type: timestamp, format: '%m/%d/%Y %H:%M:%S.%L', timezone: '+0900'}
filters:
  - type: ragnalog
    columns:
      extra: {{extra}}
out:
  type: elasticsearch
  nodes:
    - {host: {{elasticsearch.host}}, port: {{elasticsearch.port}} }
  cluster_name: {{elasticsearch.cluster_name}}
  bulk_actions: 30000
  index: {{index_name}}
  index_type: csv
