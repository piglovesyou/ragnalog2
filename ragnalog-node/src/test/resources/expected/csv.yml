in:
  type: file
  path_prefix: /tmp/ragnalog/sample.log
  parser:
    charset: UTF-8
    newline: CRLF
    type: csv
    columns:
      - {name: timestamp, type: timestamp, format: '%m/%d/%Y %H:%M:%S.%L', timezone: '+0900'}
filters:
  - type: ragnalog
    columns:
      extra: ap1
out:
  type: elasticsearch
  nodes:
    - {host: localhost, port: 9300 }
  cluster_name: ragnalog2.elasticsearch
  bulk_actions: 30000
  index: ragnalog-test
  index_type: csv
